pkg/activator/handler/testdata/config-tracing.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2019 The Knative Authors
```

Comment:
This seems vastly outdated and should be updated to the latest config-tracing.yaml from config I think?

Reply:
thanks @markusthoemmes, will update.